kornia/contrib/vit.py

```diff
@@ -233,4 +237,53 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
         out = self.patch_embedding(x)
         out = self.encoder(out)
+        out = self.norm(out)
         return out
+
+    @staticmethod
+    def from_config(variant: str, pretrained: bool = False, **kwargs: Any) -> VisionTransformer:
+        """Build ViT model based on the given config string. The format is `vit_{size}/{patch_size}`.
+        E.g. `vit_b/16` means ViT-Base, patch size 16x16. If `pretrained=True`, AugReg weights are loaded.
+        The weights are hosted on HuggingFace's model hub: https://huggingface.co/kornia.
+
+        Args:
```

Comment:
add something like this, for users now what variant's are available, and each ones have weights for it 

```suggestion
        The weights are hosted on HuggingFace's model hub: https://huggingface.co/kornia.

         .. note:: The available weights are: ...

        Args:
```